docs/data/base/getting-started/usage/usage.md

```diff
@@ -20,60 +20,71 @@ function App() {
 ```
 
 You can play around with this code in the interactive Code Sandbox demo below.
-Try importing an `InputUnstyled` component and adding it to the `<div>`:
+Try importing an [`InputUnstyled`](/base/react-input/) component and adding it to the `<div>`:
 
 {{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
 ## Shared props
 
 Base components are self-supporting and fully functional in isolation.
 
-Each component has its own unique API, but all components accept the following shared props:
+Each component has its own unique API, but all _non-utility_ components accept the following shared props:
 
 ### components
 
-The `components` prop is an object that lets you override any interior subcomponents ("slots") of the base component itself.
+The `components` prop is an object that lets you override any interior subcomponents—known as **slots**—of the base component itself.
 
 :::info
-Each component contains a `Root` slot, and many complex components (such as [`BadgeUnstyled`](/base/react-badge/)) are composed of additional subcomponents that can be customized.
-:::
+Each component contains a root slot, and other appropriate slots based on the nature of the component. For example, the `BadgeUnstyled` contains two slots:
+
+- `root`: the container element that wraps the children.
+- `badge`: the element that appears at the given position.
+  :::
 
-You can use the `components` prop to replace subcomponents with either custom components or HTML elements.
+You can use the `components` prop to override default slots with either custom components or HTML elements.
 
 For example, the [`BadgeUnstyled`](/base/react-badge/) component renders a `<span>` by default.
-The code snippet below shows how to override this by assigning a `<div>` to the `Root` element:
+The code snippet below shows how to override this by assigning a `<div>` to the root slot:
 
 ```jsx
 <BadgeUnstyled components={{ Root: 'div' }} />
 ```
 
+:::info
+If you are customizing a component like [`ButtonUnstyled`](/base/react-button/) that only has a root slot, you may prefer to use the more succinct `component` prop (described below) instead of `components`.
+:::
+
 ### component
 
 The (singular) `component` prop provides a shortcut to `components.Root`.
-This is useful if you are only overriding the `Root` element of the component.
+This is useful if you are only overriding the root element of the component.
 
-The code snippet below shows how to override the `Root` element of the [`BadgeUnstyled`](/base/react-badge/) component using the `component` prop:
+The code snippet below shows how to override the root element of the [`BadgeUnstyled`](/base/react-badge/) component using the `component` prop:
 
```

Comment:
There is a subtle difference between `component` and `components.Root`: if you override a `component`, you will be able to use props of an overridden root on the unstyled component itself. For example, if you have `<OptionUnstyled component="td">...` you will be able to use `td`'s props (such as `colspan`) on the OptionUnstyled itself. But if you use components.Root, you'll have to place these additional props on the `componentsProps.root` object.

I think this is worth mentioning somewhere.

Reply:
I have added a section on "Best practices" to address this. 👍